client/rhel/spacewalk-client-tools/src/up2date_client/rhnreg_constants.py

```diff
@@ -260,7 +258,7 @@
 
 SSL_CERT_ERROR_MSG         = _("<b><span size=\"16000\">Incompatible Certificate File</span></b>\n\n"
                                "The certificate you provided, <b>%s</b>, is not compatible with "
-                               " the Red Hat Satellite server at <b>%s</b>. You may want to double-check"
+                               " the Uyuni server at <b>%s</b>. You may want to double-check"
```

Comment:
```suggestion
                               " the Server at <b>%s</b>. You may want to double-check"
```

Since we do not have PRODUCT_NAME, please try to use something as neutral as possible (i. e. avoid "Uyuni" or "SUSE Manager"). There are some more opportunities above where I did not comment.